identity/app/controllers/EditProfileController.scala

```diff
@@ -157,6 +176,27 @@ class EditProfileController(
       } // end authActionWithUser.async
     } // end csrfCheck
 
+  private def repermissionJourneyView(
```

Comment:
this is just a shameless copypaste of profileFormsView for now, i decided to separate it because there's a chance it might end up quite different BUT if it doesn't it can be unified l8